notebooks/Customer_Automotive_Churn_Predictions.ipynb

```diff
@@ -0,0 +1,1866 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Customer_Automotive_Churn_Predictions: PyMapD- XGBoost - MapD\n",
+    "# Response Variable: Col_1"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Importing Libraries"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pymapd\n",
+    "import pygdf\n",
+    "import time\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "from copy import deepcopy\n",
+    "from sklearn.preprocessing import LabelEncoder\n",
+    "le= LabelEncoder()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Data Extaction"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Setting up MapD connection"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Connection(mapd://mapd:***@http://localhost:9090/mapd?protocol=http)\n"
+     ]
+    }
+   ],
+   "source": [
+    "dbname    = 'mapd'\n",
+    "username  = 'mapd'\n",
+    "password  = 'HyperInteractive'\n",
+    "hostname  = 'localhost'\n",
+    "mport     = 9090\n",
+    "\n",
+    "con = pymapd.connect(user=username,password=password,dbname=dbname,host=hostname,port=mport,protocol='http')\n",
+    "print(con)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Fetch data from MapD to PyGDF"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Number of Columns: 23\n"
+     ]
+    }
+   ],
+   "source": [
+    "table= 'churn'\n",
```

Comment:
Is the `churn` data public? If so, should have a section showing how to import it into MapD.

Reply:
added the URL

Reply:
Added a comment about the privacy of dataset.